src/panels/calendar/ha-big-calendar.js

```diff
@@ -0,0 +1,81 @@
+import '@polymer/app-layout/app-header-layout/app-header-layout.js';
+import '@polymer/app-layout/app-header/app-header.js';
+import '@polymer/app-layout/app-toolbar/app-toolbar.js';
+import '@polymer/paper-dropdown-menu/paper-dropdown-menu.js';
+import '@polymer/paper-icon-button/paper-icon-button.js';
+import '@polymer/paper-input/paper-input.js';
+import '@polymer/paper-item/paper-item.js';
+import '@polymer/paper-listbox/paper-listbox.js';
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { PolymerElement } from '@polymer/polymer/polymer-element.js';
+import '@vaadin/vaadin-date-picker/vaadin-date-picker.js';
+
+import React from 'react';
+import { render } from 'react-dom';
+import BigCalendar from 'react-big-calendar';
+import moment from 'moment';
+
+import '../../resources/ha-style.js';
+
+/*
+ * @appliesMixin LocalizeMixin
+ */
+
+class HABigCalendar extends PolymerElement {
+  static get template() {
+    return html`
+    <template>[[events]]</template>
```

Comment:
This makes no sense. You're also overriding the ShadowDOM

Reply:
You're right. This stuff is useless, I was using it for debugging, I forgot to remove it...